pkgs/misc/universal-android-debloater/default.nix

```diff
@@ -0,0 +1,71 @@
+{ android-tools
+, cargo
+, clang
+, cmake
+, fetchFromGitHub
+, fontconfig
+, freetype
+, lib
+, libglvnd
+, makeWrapper
+, mold
+, pkg-config
+, rustPlatform
+, rustc
+, stdenv
+, xorg
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "universal-android-debloater";
+  version = "unstable-2023-04-10";
+
+  src = fetchFromGitHub {
+    owner = "0x192";
+    repo = pname;
+    rev = "11f27c671cba278d71296cdef4c5a5dba06add5e";
+    hash = "sha256-5vGn7FRkko44rkMID23XK1YTjFBuhA4Hz8h49gB/L2Q=";
+  };
+
+  cargoLock = {
+    lockFile = "${src}/Cargo.lock";
+    allowBuiltinFetchGit = true;
+  };
+
+  nativeBuildInputs = [
+    cargo
+    clang
+    cmake
+    mold
+    pkg-config
+    rustc
+    makeWrapper
+  ];
+
+  buildInputs = [
+    fontconfig
+    freetype
+    libglvnd
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXi
+    xorg.libXrandr
+  ];
+
+  doCheck = false;
```

Comment:
tests would fail with:
```
...
 /build/cargo-vendor-dir/expat-sys-2.1.6/expat/tests/runtests.c:1419:40: warning: ISO C++ forbids converting a string constant to 'char*' [-Wwrite-strings]
   1419 |     TCase *tc_namespace = tcase_create("XML namespaces");
        |                                        ^~~~~~~~~~~~~~~~
  CMake Error at cmake_install.cmake:52 (file):
    file cannot create directory: /var/empty/local/lib.  Maybe need
    administrative privileges.
```

Reply:
Yes, indicate that on a comment above. This is just to make it clear why it is disabled.